errai-cdi/jboss/src/main/java/org/jboss/errai/cdi/server/as/JBossServletContainerAdaptor.java

```diff
@@ -34,8 +34,8 @@
   @SuppressWarnings("unused")
   private final Process jbossProcess;
 
-  private static final String NATIVE_CONTROLLER_PATH = "remote://localhost:9999";
-  private static final String HTTP_CONTROLLER_PATH = "http-remoting://localhost:9990";
+  private static String NATIVE_CONTROLLER_PATH = "remote://localhost:9999";
```

Comment:
By convention only constants use upper-case letters and underscores. Since these fields are now mutable the names should be changed to `nativeControllerPath` and `httpControllerPath`.

Also, since you assign values to these fields in the constructor below, these should probably become instance fields (i.e. not `static`).